archive/m/mojiscript/README.md

```diff
@@ -0,0 +1,16 @@
+# Sample Programs in MojiScript
+
+- hello-world.mjs
```

Comment:
I know this got merged, but the README is inconsistent with the rest of the repo. Can we revisit this?